### PR TITLE
Updated .gitignore to include LastFail picture

### PR DESCRIPTION
**Pull request by andrewzhang707**
Added LastFail picture to .gitignore so that it will not be included as a change in the Git Repo.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+#Picture Saves
+LastFail.jpg
```
Comment (prcHeavyz) [inline .gitignore:132]: Good catch

Comment (alexzhang7) [reply .gitignore:132]: Yes, looks good.

**Review by prcHeavyz — APPROVED** [review]
Good to go

**Review by alexzhang7 — APPROVED** [review]
Approved.

**Pull request merged**